src/styles/01-atoms/links/link/_link.scss

```diff
@@ -13,7 +13,7 @@ $color-link--hover: $nypl-blue-dark;
 @mixin more-link {
   display: inline-flex;
   color: $color-link;
-  text-decoration: none;
+  text-decoration: underline;
```

Comment:
Can you modify this to only be on hover?

```
text-decoration: none;

&::hover {
  text-decoration: underline;
}
```

Reply:
will having it only on hover fulfill the A11Y guideline of "Visual design always sets apart from static text and with more than color. Could be with underline or with icon" on MT-18? http://themetronome.co/components/mt-18-text-link-w-icon/?fresh=true

Reply:
I believe the icon in this instance provides the differentiation.

Reply:
WAAAAAAAAIT oh no I think I might have misunderstood the component:  

Right now I have it implemented as `unordered list of text-link-w-icon` where the bullet comes from `ul`. In this case, `text-link-with-icon` is using the `no-icon` version.  (MT-18-with-no-icon is different visually than MT-173).  I thought it was this way because in OL-32, the mockups show a list of links with underlines in un-hover state. 

It is possible, too, to implement this as "unordered list of text-link-with-icon` where the bullet is the icon that belongs to the text link.  Is that the correct way to think about this component?

Reply:
Ah. I understand the issue now.

We still shouldn't be modifying the `text-decoration` at the `more-link` level, we should be modifying a level up when wrapped in an `li`. Otherwise it will affect all `more-link`s.

Reply:
`more-link` (http://themetronome.co/components/mt-18-text-link-w-icon/?fresh=true) has the same a11y requirement.  It makes me think that text should have an underline if there is no icon, but if there is an icon, underline should be hidden ... ?

Reply:
That seems reasonable to me. I will open an issue for it on the documentation site and make the change.

Reply:
I actually decided against adding an open issue—reading over Willa's documentation I feel like she's sufficiently captured it.